epochtalk/epochtalk
# Fix auth

Test Procedure:
1) Test that moderators can only moderate users in their board, with a lower priority than moderator
2) Test that global moderators can moderate users in any board, as long as they are lower priority than a global moderator
3) Test that all moderators types only see icons for actions they can perform
4) Ensure self mod still works as expected

Fixes #664 
Fixes #685
